src/main/kotlin/blackjack/Deck.kt

```diff
@@ -0,0 +1,40 @@
+package blackjack
+
+class Deck(
+    private val cards: MutableList<Card>
+) {
+    constructor(vararg cards: Card) : this(cards.toMutableList())
+
+    fun draw(): Card {
+        return cards.removeLast()
+    }
```

Comment:
간단한 표현식으로 나타낼 수 있다면 다음과 같이 표현해줘도 좋을 것 같아요🙂
```suggestion
    fun draw(): Card = cards.removeLast()
```

Reply:
코틀린은 참 간결한게 좋네요 Single-expression function 감사합니다 😀